src/formulas/code_block.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Block of code that produces a value.
+ */
+export interface CodeBlock {
```

Comment:
maybe could be simply named `Code`? What do you think?

Reply:
I also tought about that...it works
the only thing is: it can be used for something else than the body of a function :man_shrugging: